src/content/recipes/pinia.md

```diff
@@ -0,0 +1,85 @@
+<div class="aside aside__no-top">
+
+This recipe assumes that you are using Vue 3 and Storybook >= 7.x. If you haven't upgraded yet, run the following command:
+
+```shell
+npx storybook@latest upgrade
+```
+
+</div>
+
+<RecipeHeader>
+
+How to setup Pinia and Storybook
+
+</RecipeHeader>
+
+Pinia is the recommended standard for state management in Vue. It gives you a lightweight and type-safe way to handle global state for your applications. This recipe will show you how to integrate Pinia into Storybook so that you can test your state connected components.
+
+If you want to look at a full example, look at this [awesome repo](https://github.com/chakAs3/vue3-pinia-storybook/tree/main) built by [ChakAs3](https://github.com/chakAs3).
+
+## Initialize pinia
```

Comment:
@Integrayshaun can you re-check this and capitalize Pinia in the recipe, as we're referring to an actual package and there's still some places where it's not.